src/views/v2/commerce/checkout/cart/footer/total.php

```diff
@@ -31,7 +31,7 @@
 			'<span class="tribe-tickets__commerce-checkout-cart-footer-total-label">',
 			'</span>',
 			'<span class="tribe-tickets__commerce-checkout-cart-footer-total-wrap">',
-			$total_value,
+			\TEC\Tickets\Commerce\Utils\Price::to_currency( $total_value ),
```

Comment:
We need to set the @version of this template TBD and add a @since TBD with a small description of the changes.